src/types.js

```diff
@@ -6,6 +6,34 @@
  */
 
 /**
+ * Pause
+ *
+ * @memberof types
+ * @static
+ */
+export type Frame = {
+  id: string,
+  displayName: string,
+  location: Location,
+  this: ?Object,
+  scope: ?Object
+}
+
+export type Why = {
+  type: string
+}
+
+export type Pause = {
+  frames: Frame[],
+  why: Why
+}
+
+export type Expression = {
+  id: number,
+  input: any
```

Comment:
Is the type of `input` really `any`? Or could we use one that is a bit more specific?